src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Presentation/Signature.cs

```diff
@@ -107,9 +181,9 @@ private void Initialize(bool setParameters)
             AddRange(_signatureHelpItem.SuffixDisplayParts, parts, prettyPrintedParts);
             Append(_signatureHelpItem.SuffixDisplayParts.GetFullText(), content, prettyPrintedContent);
 
-            if (_currentParameter != null)
+            if (_parameterIndex >= 0)
```

Comment:
You'll notice the previous code was not prepared to deal with IndexOf returning -1 either. An exception in that case is fine.